operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -118,13 +116,31 @@ def open(self):
     self.CUSTOM_PREFIX_WLDF = 'Sit-Cfg-WLDF--'
     self.CUSTOM_PREFIX_CFG  = 'Sit-Cfg-CFG--'
 
-    self.INTROSPECT_HOME        = '/tmp/introspect/' + self.DOMAIN_UID
-    self.TOPOLOGY_FILE          = self.INTROSPECT_HOME + '/topology.yaml'
-    self.CM_FILE                = self.INTROSPECT_HOME + '/' + self.CUSTOM_PREFIX_CFG + 'introspector-situational-config.xml'
-    self.BOOT_FILE              = self.INTROSPECT_HOME + '/boot.properties'
-    self.USERCONFIG_FILE        = self.INTROSPECT_HOME + '/userConfigNodeManager.secure'
-    self.USERKEY_FILE           = self.INTROSPECT_HOME + '/userKeyNodeManager.secure'
-    self.DOMAIN_SECRET_MD5_FILE = '/tmp/DomainSecret.md5'
+    self.INTROSPECT_HOME          = '/tmp/introspect/' + self.DOMAIN_UID
+    self.TOPOLOGY_FILE            = self.INTROSPECT_HOME + '/topology.yaml'
+    self.CM_FILE                  = self.INTROSPECT_HOME + '/' + self.CUSTOM_PREFIX_CFG + 'introspector-situational-config.xml'
+    self.BOOT_FILE                = self.INTROSPECT_HOME + '/boot.properties'
+    self.USERCONFIG_FILE          = self.INTROSPECT_HOME + '/userConfigNodeManager.secure'
+    self.USERKEY_FILE             = self.INTROSPECT_HOME + '/userKeyNodeManager.secure'
+
+    # Model in image attributes
+
+    self.DOMAIN_SECRET_MD5_FILE   = '/tmp/DomainSecret.md5'
```

Comment:
@jshum2479 Maybe for post V1:  Please consider renaming all MII-only variables so that they start with 'MII_" or maybe "MODEL_" (your choice), and maybe also do this in the shell scripts too.  That'd help differentiate the globals that are specific to MII when reading through the code.  Similarly, if adding a MII only function/method, consider decorating it with 'model' or 'mii'...

Reply:
@jshum2479

Reply:
Yes, let's leave it for post V1

Reply:
I'll file a JIRA to track.  Resolving.